example-react-app/src/i18n/en.json

```diff
@@ -70,6 +70,7 @@
   "screen.StructureComponent": "Structure Component",
   "screen.TestBlankScreen": "Test Blank Screen",
   "screen.ExampleCustomScreen": "Example Custom Screen",
+  "addons.Addons": "Addons",
```

Comment:
This key is missing in `zh-cn` file (should be added with `addons.Addons [TRANSLATION NOT AVAILABLE]` placeholder).

i18n test should fail on that, but it doesn't, because `SUPPORTED_CLIENT_LOCALES` doesn't contain `zh-cn` (#560).